lib/resources/example_json/r4_examples_care_plan.rb

```diff
@@ -0,0 +1,420 @@
lide activities&#10;Resolved&#10;Active labor&#10;Inconsistent results on voice testing&#10;Marital problems&#10;O/E - tuning fork testing&#10;Z deformity of thumb&#10;Surgical History&#10;Tone deafness (2006),  Tonsil biopsy sample (Week of 05/17/1989),  ADA - Adenosine deaminase (1989).&#10;Family History&#10;Patient was adopted&#10;Social History&#10;Alcohol&#10;Current, Beer, Wine, Liquor, Home made hard stuff, Several times per day, 5 drinks/episode average. 10 drinks/episode maximum. Started age 8 Years. Previous treatment: None. Alcohol use interferes with work or home: Yes. Drinks more than intended: Yes. Others hurt by drinking: No. Ready to change: No. Household alcohol concerns: No.&#10;Employment/School&#10;Employed, Work/School description: Desk Job. Activity level: Desk/Office. Operates hazardous equipment: No.&#10;Exercise&#10;Exercise duration: 30. Exercise frequency: 3-4 times/week. Self assessment: Fair condition. Exercise type: Walking, Swimming.&#10;Home/Environment - Low Risk&#10;Lives with Alone. Living situation: Home/Independent. Alcohol abuse in household: Yes. Substance abuse in household: Yes. Smoker in household: No. Injuries/Abuse/Neglect in household: No. Feels unsafe at home: No. Safe place to go: Yes. Agency(s)/Others notified: No. Family/Friends available for support: Yes. Concern for family members at home: No. Major illness in household: No. Financial concerns: No. TV/Computer concerns: No. Risks in environment: Does not wear helmet, Pets/Animal exposure, Stairs.&#10;Nutrition/Health&#10;Type of diet: Generl. Wants to lose weight: No. Sleeping concerns: No. Feels highly stressed: Yes.&#10;Sexual&#10;Sexually active: No. History of sexual abuse: No.&#10;Substance Abuse - Medium Risk&#10;Current, Amphetamines, Cocaine, Ecstasy, Hallucinogens/LSD, Heroin, Inhalants/Glues/Solvents, Marijuana, Methamphetamines, Prescription medications, Daily, Previous treatment: Treatment center, Inpatient, Outpatient. Started age 8 Years. IV drug use: Yes. Drug use interferes with work/home: Yes. Ready to change: No. Household substance abuse concerns: No.&#10;Tobacco&#10;Current every day smoker, Cigars, hand, Household tobacco concerns: Yes&#10;Allergies&#10;oxyCODONE&#10;Peanuts&#10;sulfa drugs&#65533;&#65533;(Anaphylactic reaction)&#10;uranium&#10;iodine&#65533;&#65533;(Hives, Sneezing)&#10;penicillin V potassium&#65533;&#65533;(Hives)&#10;skin test antigens, multiple&#10;Home Medication List&#10;Aceon 2 mg oral tablet, 1 tab(s), Oral, Daily, 11 refills&#10;acetaminophen-tramadol&#10;Physical Exam&#10;Vitals &amp; Measurements&#10;Lab Results&#10;Radiology Results&#10;Assessment/Plan&lt;/p&gt;&lt;p&gt;&lt;b&gt;Status&lt;/b&gt;: Active&lt;/p&gt;&lt;p&gt;&lt;b&gt;Intent&lt;/b&gt;: Plan&lt;/p&gt;&lt;p&gt;&lt;b&gt;Category&lt;/b&gt;: Assessment and Plan of Treatment&lt;/p&gt;&lt;p&gt;&lt;b&gt;Author&lt;/b&gt;: Shaik, Faiz Ahmed&lt;/p&gt;&lt;p&gt;&lt;b&gt;Effective Period&lt;/b&gt;: Mar 10, 2016  7:43 P.M. UTC&lt;/p&gt;&lt;/div&gt;"
```

Comment:
Can we avoid formatting the div converting the # into html entity?

Reply:
Resolved